src/GafferArnold/ArnoldDisplacement.cpp

```diff
@@ -200,7 +205,10 @@ bool ArnoldDisplacement::acceptsInput( const Gaffer::Plug *plug, const Gaffer::P
 	{
 		if( const GafferScene::Shader *shader = runTimeCast<const GafferScene::Shader>( inputPlug->source<Plug>()->node() ) )
 		{
-			return runTimeCast<const ArnoldShader>( shader );
+			return
```

Comment:
Same braces comment as the other PR. Now you've got me second guessing whether that's our convention or I just haven't had enough coffee yet...